pkg/migration/controllers/migrationschedule.go

```diff
@@ -107,7 +111,89 @@ func (m *MigrationScheduleController) handle(ctx context.Context, migrationSched
 
 		return nil
 	}
+	migrationSchedule.Spec = setScheduleDefaults(migrationSchedule.Spec)
+	if migrationSchedule.GetAnnotations() != nil {
+		if _, ok := migrationSchedule.GetAnnotations()[StorkMigrationAnnotation]; ok {
+			// check status of all migrated app in cluster
+			logrus.Infof("Migration schedule is on dr cluster")
+			isActivated, err := getMigratedAppStatus(migrationSchedule)
+			if err != nil {
+				return err
+			}
+			migrationSchedule.Status.ApplicationActivated = isActivated
+			msg := fmt.Sprintf("Setting AppActive status to: %v", isActivated)
+			m.recorder.Event(migrationSchedule,
+				v1.EventTypeWarning,
+				"AppsActivated",
+				msg)
+			log.MigrationScheduleLog(migrationSchedule).Warn(msg)
+			return m.client.Update(context.TODO(), migrationSchedule)
+		}
+	}
+	if !(*migrationSchedule.Spec.Suspend) {
+		remoteConfig, err := getClusterPairSchedulerConfig(migrationSchedule.Spec.Template.Spec.ClusterPair, migrationSchedule.Namespace)
+		if err != nil {
+			return err
+		}
+		remoteOps, err := storkops.NewForConfig(remoteConfig)
+		if err != nil {
+			m.recorder.Event(migrationSchedule,
+				v1.EventTypeWarning,
+				string(stork_api.MigrationStatusFailed),
+				err.Error())
+			return nil
+		}
+		coreOps, err := core.NewForConfig(remoteConfig)
+		if err != nil {
+			m.recorder.Event(migrationSchedule,
+				v1.EventTypeWarning,
+				string(stork_api.MigrationStatusFailed),
+				err.Error())
+			return nil
+		}
+		migrSched, err := remoteOps.GetMigrationSchedule(migrationSchedule.Name, migrationSchedule.Namespace)
+		if errors.IsNotFound(err) {
+			// TODO:generate event?
+			namespace, err := core.Instance().GetNamespace(migrationSchedule.Namespace)
+			if err != nil {
+				return err
+			}
+			namespace.ResourceVersion = ""
+			_, err = coreOps.CreateNamespace(namespace)
+			if err != nil && !errors.IsAlreadyExists(err) {
+				return err
+			}
+			// create new
+			remoteMigrSched := migrationSchedule.DeepCopy()
+			remoteMigrSched.ResourceVersion = ""
+			if remoteMigrSched.Annotations == nil {
+				remoteMigrSched.Annotations = make(map[string]string)
+			}
+			remoteMigrSched.Annotations[StorkMigrationAnnotation] = "true"
```

Comment:
We should add an annotation indicating that this is a copy of the MigrationSchedule from the source cluster since it will be pointing to a CLusterPair which does not exist. We need some way to indicate that this is a "non-usable" MigrationSchedule. They cannot expect migrations to start happening automatically in the reverse direction.

We could also rename it to `static-copy-<name of migration schedule on source>`.

Reply:
make sense, updated annotation name